src/cmd/cos_customizer/run_script.go

```diff
@@ -87,6 +92,18 @@ func createEnvFile(prefix string, files *fs.Files, env map[string]string) (strin
 	return filepath.Base(envFile.Name()), nil
 }
 
+// createEnvString creates an environment variable string used by the
+// provisioner tool. The format is the same as the format used by exec.Command.
+// Elements are sorted for predictable output.
+func createEnvString(m map[string]string) string {
+	var elems []string
+	for k, v := range m {
+		elems = append(elems, k+"="+v)
+	}
+	sort.Strings(elems)
```

Comment:
nice! map iteration in golang is not guaranteed to be same across runs, this sorting will help keep a predictable order.